BitmovinConvivaAnalytics/Classes/ConvivaAnalytics.swift

```diff
@@ -366,7 +382,7 @@ extension ConvivaAnalytics: BitmovinPlayerListenerDelegate {
 
     func onError(_ event: ErrorEvent) {
         if !isSessionActive {
-            internalInitializeSession()
+            internalInitializeSession(fromPlay: false)
```

Comment:
Ah that's why we maybe need this flag 🤔 What is the difference when an error happened? 🤔

Reply:
I don't really get this. You call `internalInitializeSession(fromPlay: false)` if no session is active; however, in `internalInitializeSession` you do an early return if no session is active and `fromPlay` is `false`. 
The only thing that happens in this case is `buildContentMetadata()`. I don't see how this is useful here, but if that's the purpose let's call `buildContentMetadata()` directly and remove this whole `fromPlay` handling.

Am I missing something?